api/v1beta1/service_types.go

```diff
@@ -49,11 +49,11 @@ type SecretKeyReference struct {
 	Key string `json:"key"`
 }
 
-// Specifies the desired state of the Service resource, i.e. the authencation/authorization scheme to be applied to protect the matching HTTP services.
```

Comment:
It's good that "HTTP services" was removed, it was a bit limited and wasn't representing the full spectrum of where Authorino is applied 👍🏼